etl/src/main/scala/org/astraea/etl/Reader.scala

```diff
@@ -103,7 +103,18 @@ object Reader {
 
   def createSchema(cols: Map[String, DataType]): StructType = {
     var userSchema = new StructType()
-    cols.foreach(col => userSchema = userSchema.add(col._1, col._2.value))
+    // TODO astraea #1286 Need to wrap non-nullable type with optional.
+    cols.foreach(col =>
```

Comment:
可以把檢查直接放進來就好，例如：

```scala
  def createSchema(cols: Map[String, DataType]): StructType = StructType(
    cols.map { case (name, t) =>
      // TODO astraea #1286 Need to wrap non-nullable type with optional.
      if (t != DataType.StringType)
        throw new IllegalArgumentException(
          "Sorry, only string type is currently supported.Because a problem(astraea #1286) has led to the need to wrap the non-nullable type."
        )
      StructField(name, t.sparkType)
    }.toSeq
  )
```

其中`sparkType`是還沒實作的方法，請放到我們自己的DataType中

Reply:
好，已修改完畢